jme3-networking/src/main/java/com/jme3/network/base/DefaultServer.java

```diff
@@ -287,7 +287,7 @@ public boolean hasConnections()
     @Override
     public Collection<HostedConnection> getConnections()
     {
-        return Collections.unmodifiableCollection((Collection<HostedConnection>)connections.values());
+        return Collections.unmodifiableCollection(connections.values());
```

Comment:
So nice that this cast isn't required anymore.

Reply:
Agreed.